### PR TITLE
Added funder facet

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/vue/components/search/ResultSet.vue [hunk 1]
```diff
@@ -38,7 +38,8 @@
                     <div class="card card-filter">
                         <Facet :facet-info="results.facetInfo.statusInfo" :facet-name="facetLabels.status" v-on="$listeners" :formValues="formValues"/>
                         <Facet :facet-info="results.facetInfo.fundingCycleInfo" :facet-name="facetLabels.fundingCycle" v-on="$listeners" :formValues="formValues" v-show="showFundingCycleFacet"/>
-                        <Facet :facet-info="results.facetInfo.fundingOrgInfo" :facet-name="facetLabels.fundingOrg" v-on="$listeners" :formValues="formValues" v-show="showFundingOrgFacet"/>
+                        <Facet :facet-info="results.facetInfo.funderInfo" :facet-name="facetLabels.funder" v-on="$listeners" :formValues="formValues"/>
+                        <!-- <Facet :facet-info="results.facetInfo.fundingOrgInfo" :facet-name="facetLabels.fundingOrg" v-on="$listeners" :formValues="formValues" v-show="showFundingOrgFacet"/> -->
```
Comment (praneethpr) [inline assets/js/vue/components/search/ResultSet.vue:42]: Should I go ahead and remove all funding Org related code?

File: src/Util/Search.php [hunk 2]
```diff
@@ -486,6 +491,56 @@ private function getProjectDirectorInfo(array $aggregations): array
...
+        foreach ($funders as $funder) {
+            $fundersInfo[$funder->getId()] = array(
+                'id' => $funder->getId(),
+                'name' => $funder->getName(),
+                'count' => $aggregations[$funder->getId()]
+            );
+        }
+        //Sorting based on highest count
+        $array_column1 = array_column($fundersInfo, 'count');
+        $array_column2 = array_column($fundersInfo, 'name');
```
Comment (mickel1138) [inline src/Util/Search.php:534]: Maybe these could use better names, but it's not too important.

File: config/packages/fos_elastica.yaml [hunk 3]
```diff
@@ -152,6 +152,11 @@ fos_elastica:
                     property_path: false
                     type: 'date'
                     format: 'yyyy-MM-dd'
+                funders:
+                    type: 'nested'
+                    properties:
+                        id:
```
Comment (fightingtexasaggie) [inline config/packages/fos_elastica.yaml:158]: Do we need to index name as well?

File: src/Util/Search.php (hunk 2 above)
Comment (praneethpr) [reply src/Util/Search.php:482]: Yes, but i felt this was anyways going to be replaced with multi-search

File: config/packages/fos_elastica.yaml (hunk 3 above)
Comment (praneethpr) [reply config/packages/fos_elastica.yaml:158]: No, we don't need to index name, because we are searching based on `id`

**Review by mickel1138 — APPROVED** [review]

**Pull request merged**